code/modules/mob/transform_procs.dm

```diff
@@ -476,25 +463,26 @@
 
 	qdel(src)
 
-/mob/living/carbon/human/AIize(transfer_after = TRUE, client/preference_source)
-	if (notransform)
-		return
-	for(var/t in bodyparts)
-		qdel(t)
-
-	return ..()
-
-/mob/living/carbon/AIize(transfer_after = TRUE, client/preference_source)
-	if (notransform)
-		return
+//A common proc to start an -ize transformation
+/mob/living/carbon/proc/pre_fooize(delete_items = FALSE)
```

Comment:
```suggestion
/mob/living/carbon/proc/pre_transform(delete_items = FALSE)
```
what does fooize even mean
how does someone who didn't look at this specific pr gonna know what this means?

Reply:
I renamed the proc and all mentions to it to pre_transform().